clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -48,19 +50,31 @@ const ScanResultsForm = () => {
   const features = useFeatures();
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
+  const { successAlert } = useAlert();
+  const { handleError } = useAPIHelper();
 
   const confirmRegisterSelectedSystems = async () => {
     dispatch(chooseSystemsForReview(selectedSystems.map((s) => s.fides_key)));
-    await upsertSystems(selectedSystems);
-
-    const datamapRoute = resolveLink({
-      href: "/datamap",
-      basePath: "/",
-    });
-
-    return features.plus
-      ? router.push(datamapRoute.href)
-      : router.push("/system");
+    const response = await upsertSystems(selectedSystems);
+
+    if (isErrorResult(response)) {
+      return handleError(response.error);
+    } 
+      const datamapRoute = resolveLink({
+        href: "/datamap",
+        basePath: "/",
+      });
+
+      successAlert(
+        `You have successfully added ${selectedSystems?.length} systems to your Data Map`,
+        `${selectedSystems?.length} Systems successfully added to your Data Map`,
+        { isClosable: true }
+      );
```

Comment:
I'm pretty certain this is not going to work for the datamap view because it's technically a different Nextjs app which the toast won't know about. I think the only way to navigate to the other app while also showing a toast might be to pass a query param, or maybe something else sneaky in `window.location`. Maybe our resident inter-zone navigation expert @ssangervasi has an idea

Reply:
Oh, interesting. I'll look into this more.

Reply:
Yeah, a query parameter is a pretty classic way of triggering something like this on a redirect. That would Plus would need to check its router on page load, trigger the alert with that text, and clear the query parameter.

One other way we have of communicating between the apps is with localStorage, as of https://github.com/ethyca/fidesctl-plus/pull/303. That's more complicated, though, so probably not worth it for this case.

And then I have one hack idea which is to simply delay the `router.push` for ~1 second, so the toast is visible before the page changes and it goes away. Not quite the experience we're designing for, though.

Reply:
Gotcha. I'll work on the plus updates as well in conjunction with this and [238](https://github.com/ethyca/fides/pull/1693).